lib/person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_reader :name, :interests, :supplies, :person
+  def initialize(person)
+    @person = person
+    @name = person[:name]
+    @interests = person[:interests]
+    @supplies = {}
+  end
```

Comment:
I don't think we need to store the entire hash as a @person here since we are building a person via `name`, `interests`, and `supplies`. So I would remove that

```suggestion
  attr_reader :name, :interests, :supplies
  def initialize(person)
    @name = person[:name]
    @interests = person[:interests]
    @supplies = {}
  end
```